definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -23,6 +23,7 @@
     "PostConditionState": "*",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/make-service-decision",
     "RetriesTimeoutURLAboutToSubmitEvent": "30,30",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "ShowSummary":  "Y"
   }
```

Comment:
Activating CYA functionality

Reply:
I don't think you need this here. I think the one you need is the one in the CaseEventToFields file

Reply:
`ShowSummary` is needed on the CaseEvent as part of the CYA set up, if `ShowSummaryChangeOption`  is added only in the CaseEventToFields and not specified here, CCD would not process it.